keyboards/keychron/k6/keymaps/ansi/keymap.c

```diff
@@ -23,58 +23,65 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 // entirely and just use numbers.
 enum layer_names {
```

Comment:
we have standardized the names in this convention across the board. Please consider matching it
https://github.com/SonixQMK/qmk_firmware/blob/sn32_master/keyboards/keychron/k12/keymaps/ansi/keymap.c#L23-L28